support/gathering-cluster-data.adoc

```diff
@@ -36,7 +36,9 @@ include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
 == Additional resources
 
```

Comment:
See comment on line 143 of the topic map. I don't see this assembly included.

Reply:
I learned after the fact that the assembly didn't need to be included. I think it OK to leave the `ifdef`s.